faster_whisper/transcribe.py

```diff
@@ -142,8 +147,7 @@ def __init__(
         self.use_vad_model = use_vad_model
         self.vad_onset = 0.500
         self.vad_offset = 0.363
-        self.vad_model_url = "https://whisperx.s3.eu-west-2.amazonaws.com/model_weights/segmentation/0b5b3216d60a2d32fc086b47ea8c67589aaeb26b7e07fcbe620d6d0b83e209ea/pytorch_model.bin"
-
+        self.vad_model_url = os.path.join(vad_url_dir, "pytorch_model.bin") 
```

Comment:
Where the model is downloaded? vad_model_url is a file path, no? Why does it say URL then? I would suggest renaming the variables to better reflect the purpose.

Reply:
The URL is pointing to the remote file path that needs to be downloaded with self.load_vad_model()

Reply:
So what is the purpose of rebuilding the URL? It's not like you can get another model from vad_url_dir considering that the path has model hash in it.

Reply:
If you goal is to reduce line length then do something like

```python
vad_model_url = (
    "https://whisperx.s3.eu-west-2.amazonaws.com/model_weights"
    "/segmentation/0b5b3216d60a2d32fc086b47ea8c67589aaeb26b7e07fcbe620d6d0b83e209ea/pytorch_model.bin"
)
```

Much easier to read and understand.